src/components/MapContainer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Map from './Map'
+import { connect } from 'react-redux'
+
+class MapContainer extends React.Component {
+	render() {
+		return (
+			<Map className="map-container" config={this.props.config}/> 
```

Comment:
Yea when I remove the className from <Map/> it removes the entire map. 
Should I pass the MapContainer's className onto Map?